build-system/tasks/e2e/index.js

```diff
@@ -128,7 +128,7 @@ async function fetchCoverage_(outDir) {
   const zipFilename = path.join(outDir, 'coverage.zip');
   const zipFile = fs.createWriteStream(zipFilename);
 
-  await new Promise((resolve, reject) => {
+  await /** @type {Promise<void>} */ (new Promise((resolve, reject) => {
```

Comment:
I wish TS did a better job with Promises. (I can't think of a better way to fix this error.)

Reply:
I really wish that as well, the without the ability to template them it gets very awkward.